projects/vamp-plugins.org/package.yml

```diff
@@ -0,0 +1,47 @@
+distributable: ~
+
+versions:
+  url: https://vamp-plugins.org/develop.html
+  match: /vamp-plugin-sdk-\d+\.\d+\.\d+\.tar\.gz/
+  strip:
+    - /^vamp-plugin-sdk-/
+    - /\.tar\.gz$/
+
+dependencies:
+  xiph.org/flac: ^1.4
+  xiph.org/ogg: ^1.3
+  github.com/libsndfile/libsndfile: ^1.2
+
+build:
+  dependencies:
+    gnu.org/automake: '*'
+    curl.se: '*'
+  script:
+    # get archive url
+    - DIST_URL=$(curl -s https://vamp-plugins.org/develop.html | grep -o 'https://code.soundsoftware.ac.uk/attachments/download/[0-9]*/vamp-plugin-sdk-[0-9.]*.tar.gz')
```

Comment:
how does this download the specific version we want?

I’m confused why we can't use the `distributable` key.

Reply:
> how does this download the specific version we want?
> 
> I’m confused why we can't use the `distributable` key.

I'm sorry, it's my inattention
I fixed it
But still we have to find this url on the page. Because it contains characters (number) that are not related to the version

Reply:
For example: `https://code.soundsoftware.ac.uk/attachments/download/2588/vamp-plugin-sdk-2.9.0.tar.gz`

we need to find this part `/2588/`

Do we have a way to do this in `distributable`?

Reply:
Right I see. No. We don’t.

So yeah you'll have to do the curl line, but you need to at least fetch the right `version`.

Reply:
It seems to me that this version will be correct.

https://github.com/pkgxdev/pantry/commit/fb6678fa0abcebd1b3c860b487c2839eab05201d